### DISCUSSION TITLE
Backend module: Which functionality do you use?

### DISCUSSION THREAD

**Discussion by oliverklee** [oliverklee/ext-seminars#1379] [Ideas]
As part of the [compatibility with TYPO3 V11](https://github.com/oliverklee/ext-seminars/wiki/11LTS-Crowdfunding-Campaign), I will completely rewrite the backend module. For this, I plan to only implement the functionality and hooks that are actually still used/needed. For this, I need your input. :-)

If you are using the backend module, please comment:

- Which of the existing functionality of the backend module do you use (or plan to use)?
- Which of the displayed data in the backend module do you actually need?
- Which of the existing hooks of of the backend module do you use? For which purpose?

**Comment by lisardo2** [comment]
Hello, 
we dont use the backend module extensivly, mostly only the CSV-Export. Working in the list module is mor convinient.

**Comment by peerlight** [comment]
## Which of the existing functionality of the backend module do you use (or plan to use)?

- Tab Events
  - edit, show/hide, show/export participants as CSV
  - feature request: button to copy event without duplicating registrations
  - feature request: needs an event filter (organizer, categories, accreditation_number, leaders, tutors) and search field for title
  - feature request: individual field select for CSV export (or multiple templates -> short/complete list ...)
  - feature request: configurable date range and organizer for filtered CSV export
- Tab Registrations
  - not used
  - feature request: needs an event filter or search field for name and e-mail
- Tabs Organizers and Speakers not used
- page tree should be still visible (to select folders of different years)

## Which of the displayed data in the backend module do you actually need?

- Tab Events: all

## Which of the existing hooks of of the backend module do you use? For which purpose?

- none

**Reply by jpmschuler** [reply]
As we just have the most basic use of fe_login, select event (no use of actual dates, prices or anything), add notes (no fields except a free field) and join waiting queue, thus:

## Which of the existing functionality of the backend module do you use (or plan to use)?
 * Tab Events
   * overview of full/waiting list
   * CSV export
   * edit record (sometimes also list view) to e.g. delete attendees which are not eligible
 * other tabs
   * as we have all events of an organisation on the same page a tab like `Registrations` is useless, as there are just hundreds of entries
   * Speakers and Organizers are done via list view

## Which of the displayed data in the backend module do you actually need?
 * Tab Events
   * title and capacity (min/max/full/waiting/enough)

## Which of the existing hooks of of the backend module do you use? For which purpose?
 * none

**Comment by oliverklee** [comment]
@peerlight Thanks! Concerning the feature requests, could you please open tickets for those (one per feature request)? Thanks!

**Reply by peerlight** [reply]
https://github.com/oliverklee/ext-seminars/issues/1385
https://github.com/oliverklee/ext-seminars/issues/1384
https://github.com/oliverklee/ext-seminars/issues/1383
https://github.com/oliverklee/ext-seminars/issues/1386

**Comment by oliverklee** [comment]
From a customer:
- list events (with pagination as a new feature)
- create event
- create location/hotel
- create a registration
- list attendees by event with the following data:
  - first name and last name (or full name), company, seats, status (waiting list?), internal comments, date of last change
  - sort by status
  - edit the linked user
- download attendee lists for event as CSV
- email to all attendees
- email to confirmed registrations (i.e., excluding the waiting list)

**Comment by oliverklee** [comment]
From a customer:
- events list view: title, date, icon
- dynamic event icons (hidden, date-controlled)
- CSV-export registrations for an event
- CSV-export events
- delete registration
